src/actions/make/make_cpp.lua

```diff
@@ -257,10 +257,8 @@
 					for cfg in project.eachconfig(prj) do
 						local filecfg = incfg[cfg]
 						if filecfg then
-							-- if the custom build outputs an object file, add it to
-							-- the link step automatically to match Visual Studio
 							local output = project.getrelative(prj, filecfg.buildoutputs[1])
-							if path.isobjectfile(output) then
+							if path.isobjectfile(output) and (filecfg.linkbuildoutputs == true or filecfg.linkbuildoutputs == nil) then
```

Comment:
I thought it occurred in the codebase all over the place, but it doesn't, ignore me. Hopefully, this is the last bad suggestion I make on this PR, haha.